### ISSUE TITLE
Error: Merkle root from the diff doesn't match calculated merkle root after diff is applied

### ISSUE THREAD

**Issue by dashameter** [dashpay/platform#112]
During registering contracts encountered error:

```
Error: Merkle root from the diff doesn't match calculated merkle root after diff is applied
    at SimplifiedMNList.applyDiff (dashpay-wallet/node_modules/@dashevo/dashcore-lib/lib/deterministicmnlist/SimplifiedMNList.js:85:11)
    at SimplifiedMasternodeListProvider.updateMasternodeList (dashpay-wallet/node_modules/@dashevo/dapi-client/lib/SimplifiedMasternodeListProvider/SimplifiedMasternodeListProvider.js:59:29)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async SimplifiedMasternodeListProvider.getSimplifiedMNList (dashpay-wallet/node_modules/@dashevo/dapi-client/lib/SimplifiedMasternodeListProvider/SimplifiedMasternodeListProvider.js:34:7)
    at async SimplifiedMasternodeListDAPIAddressProvider.getLiveAddress (dashpay-wallet/node_modules/@dashevo/dapi-client/lib/dapiAddressProvider/SimplifiedMasternodeListDAPIAddressProvider.js:21:17)
    at async GrpcTransport.request (dashpay-wallet/node_modules/@dashevo/dapi-client/lib/transport/GrpcTransport/GrpcTransport.js:46:21)
    at async PlatformMethodsFacade.broadcastStateTransition (dashpay-wallet/node_modules/@dashevo/dapi-client/lib/methods/platform/broadcastStateTransition/broadcastStateTransitionFactory.js:33:50)
```

**Comment by mayoreee** [comment]
Related to #244

**Comment by shumkov** [comment]
I've not seen this problem for a while. I think it fixed with #1859
